ntoskrnl/se/token.c

```diff
@@ -81,6 +81,32 @@ static const INFORMATION_CLASS_INFO SeTokenInformationClass[] = {
 
 /* FUNCTIONS *****************************************************************/
 
+/**
+ * @brief
+ * Determines whether two token sources are equal.
+ *
+ * @param[in] FirstTokenSource
+ * The first token source.
+ *
+ * @param[in] SecondTokenSource
+ * The second token source.
+ *
+ * @return
+ * Returns TRUE if both token sources are equal amongst
+ * their properties, FALSE otherwise.
+ */
+static
+BOOLEAN
+SepEqualTokenSource(
```

Comment:
Well, I'd say not all functions need descriptions. This one is just wasting 13 lines of space :)

Reply:
So you're just saying I can totally remove the documentation header for this function altogether?